### ISSUE TITLE
[NF] Add possibility to calculate duration time considering parallelization

### ISSUE THREAD

**Issue by eugenenekh** [aquality-automation/aquality-tracking#125]
As an aquality-tracking user
I want to see actual run duration
So that I don't need to go into CI and check e.g. 'threadCount' parameter and calculate duration myself

Right now, as I understand, aquality tracking calculates total run duration as a sum of all tests duration. This obviously does not consider parallelization. So probably we could add a parameter, like 'threadCount', during import (both for ui and api imports, optional, default is 1) and use it in duration calculations?

**Comment by mialeska** [comment]
>aquality tracking calculates total run duration as a sum of all tests duration.

Sounds wrong. Total run duration should be calculated as the duration between **the earliest test start** time and **the latest test finish time**. When it is calculated this way, there will be no need to use any additional parameters

**Comment by eugenenekh** [comment]
@mialeska, @DmitryBogatko, @runetsk 
I've investigated a bit:

Default `JSONFormatter` in cucumber versions prior to 4.3.0 **has no timestamp for test start**. That's why we have following logic implemented here:
1. set run finish time as time when we upload cucumber.json report (which smells a bit)
2. calculate duration as a sum of durations for all tests/steps
3. set run start time as finish time minus total duration

And all this eventually breaks as soon as we add another thread...

From cucumber version 4.3.0 `start_timestamp` parameter was added:
https://github.com/cucumber/cucumber-jvm/blob/cf80dedbf1e9f143b792f4413b9abb912653d43f/core/src/main/java/cucumber/runtime/formatter/JSONFormatter.java#L197

If we do **not** want to break anything in terms of backward compatibility, we still need to play with such additional parameter for cucumber with versions below 4.3.0. For versions 4.3.0+ we can add this  `start_timestamp` parameter to `ScenarioDto` and refactor duration calculation logic like mentioned above.
